app/events/definitions/IFederationInfo.ts

```diff
@@ -0,0 +1,3 @@
+export interface IFederationInfo {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [An empty interface is equivalent to `{}`.](https://app.codacy.com/app/RocketChat/Rocket-Chat/pullRequest?prid=4420492)